.env.server.example

```
@@ -1,15 +1,13 @@
-# Node captcha verify server port
+﻿# Local Node captcha verify server port
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# Local Node captcha verify server port key has incorrect delimiter

(IncorrectDelimiter)